dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoDualWriterClient.java

```diff
@@ -191,5 +244,535 @@ public String setex(final String key, int seconds, String value) {
         return targetClient.setex(key, seconds, value);
     }
 
+    @Override
+    public Long del(final String key) {
```

Comment:
This is applicable to all operations being implemented here. Both del and d_del are public and we wont get the dual write functionality if someone calls d_ variants for an operation.
Correct me if I am wrong, but you should call d_ variant of the dual writer here. And that guy should implement the dual writer.